lib/subproviders/geth_api_double.js

```diff
@@ -164,7 +164,7 @@ GethApiDouble.prototype.eth_hashrate = function(callback) {
   callback(null, "0x0");
 };
 
-GethApiDouble.prototype.eth_gasPrice = function(callback) {
+GethApiDouble.prototype.eth_gasPrice = function(callback, address = "0xabc") {
```

Comment:
`undefined` might be better to make sure no one mistakes `0xabc` as a valid value.
```suggestion
GethApiDouble.prototype.eth_gasPrice = function(callback, address = undefined) {
```